### DISCUSSION TITLE
Adding data to another table

### DISCUSSION THREAD

**Discussion by erakastrati** [Budibase/budibase#2347] [Help]
I am facing some problems with the tables.
Check the first photo: I am doing an application for a furniture shop that also has a section to manage the tasks of the employees. After you click 'Add project' (first photo), it should send you to Project breakdown, where the project is broken down in small pieces, whatever, that is a special request. 
And the request after that is that if you hit 'shto tek moduli'(second photo), that row should be added to another table(3d photo), where that error is shown there. the links to swap between table to table are OK, I don't really know the problem.
I would be happy if you could help me.
Thanks a lot for your time,
Era.

<img width="1348" alt="Screen Shot 2021-08-12 at 3 36 08 PM" src="https://user-images.githubusercontent.com/74623805/129216200-4bed9903-3b0d-402c-b8dd-2a24c6453050.png">

<img width="1328" alt="Screen Shot 2021-08-12 at 3 51 10 PM" src="https://user-images.githubusercontent.com/74623805/129216273-2c7fd087-6b37-4996-8039-867779919fb4.png">

<img width="1388" alt="Screen Shot 2021-08-12 at 3 53 31 PM" src="https://user-images.githubusercontent.com/74623805/129216399-0755350d-7cd1-4474-acf5-92ac718810f9.png">

**Comment by joebudi** [comment]
Hey @erakastrati 

I believe you got in touch with @mjashanks and he has agreed to chat with you. I'll leave this open and it would be great if you could add the solution to your problem here so others may benefit from it.

Thanks, Joe

**Reply by erakastrati** [reply]
Hi there Joe.
I got in touch with him yesterday and he was on leave day. He said that we can book a call next week, but I actually need it faster. Is someone else from the team free before next week?
